README.md

```diff
@@ -106,22 +106,43 @@ Async do
 	internet = Async::HTTP::Internet.new
 	barrier = Async::Barrier.new
 	semaphore = Async::Semaphore.new(2, parent: barrier)
-	
+
 	# Spawn an asynchronous task for each topic:
 	TOPICS.each do |topic|
 		semaphore.async do
 			response = internet.get "https://www.google.com/search?q=#{topic}"
 			puts "Found #{topic}: #{response.read.scan(topic).size} times."
 		end
 	end
-	
+
 	# Ensure we wait for all requests to complete before continuing:
 	barrier.wait
 ensure
 	internet&.close
 end
 ```
 
+### Accept Encoding Headers
+
+`Async::HTTP` automatically handles gzip encoding as of `v0.56.0`, e.g.
+
+``` ruby
+#!/usr/bin/env ruby
+
+require 'async'
+require 'async/http/internet'
+
+Async do
+  internet = Async::HTTP::Internet.new
+  url = "http://httpbin.org/gzip"
+  headers = ["accept-encoding", "gzip"] # or {"accept-encoding" => "gzip"}
```

Comment:
The problem is it should be an Array of Array(key, value).

Similar to `{'accept-encoding': 'gzip'}.to_a => [[:"accept-encoding", "gzip"]]`

Reply:
Ah, of course: fixed in 72ce63b